kube-hunter.py

```diff
@@ -35,9 +35,12 @@
 
 from src.modules.report.plain import PlainReporter
 from src.modules.report.yaml import YAMLReporter
+from src.modules.report.json_reporter import JSONReporter
```

Comment:
Could we call the module file `json` rather than `json_reporter` for consistency with the other two?

Reply:
Done